test/src/test/java/hudson/model/MyViewTest.java

```diff
@@ -63,7 +63,7 @@ public void testContains() throws IOException, Exception{
         
         GlobalMatrixAuthorizationStrategy auth = new GlobalMatrixAuthorizationStrategy();   
         rule.jenkins.setAuthorizationStrategy(auth);
-        User user = User.get("User1");
+        User user = User.getOrCreateByIdOrFullName("User1");
```

Comment:
Is this related to the change? Or is it something you just oversaw during the development?

Reply:
No, it isn't. Its just a minor cleanup for deprecated functions. Its functionally equivalent